examples/complex_fluids/ex0/example.cpp

```diff
@@ -36,10 +23,19 @@
 #include <ibtk/muParserCartGridFunction.h>
 #include <ibtk/muParserRobinBcCoefs.h>
 
+#include <petscsys.h>
+
+#include <BergerRigoutsos.h>
+#include <CartesianGridGeometry.h>
+#include <LoadBalancer.h>
+#include <SAMRAI_config.h>
```

Comment:
```suggestion
```
We never need to include this header ourselves - might as well get rid of it. Can you just delete this inclusion now here and the other files instead of putting it in the reorganized lists?